src/transaction_processor.rs

```diff
@@ -572,20 +573,31 @@ impl<FG: ForkGraph> TransactionBatchProcessor<FG> {
             error_counters.invalid_compute_budget += 1;
         })?;
 
-        let fee_payer_address = message.fee_payer();
+        let mut fee_payer_address = *message.fee_payer();
 
-        let mut loaded_fee_payer = match account_loader.load_account(fee_payer_address, true) {
+        let mut loaded_fee_payer = match account_loader.load_account(&fee_payer_address, true) {
             Some(account) => account,
```

Comment:
what if the feepayer is found but not delegated? then the transaction will fail post execution, although the intention was that the escrow should be charged.